scripts/visualizations/config.xsl

```diff
@@ -2,7 +2,6 @@
 <xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="1.0">
 
     <xsl:variable name="siteroot">localhost:3000</xsl:variable>
-    <xsl:variable name="searchroot">http://cdrhsearch.unl.edu:8080/solr/austen/select/?version=2.2&amp;indent=on&amp;echoParams=all</xsl:variable>
```

Comment:
I didn't see it being used in `chapter_generations.xsl` anymore, so I just ditched it, especially since cdrhsearch isn't a thing anymore.  Not really sure what the chapter_generation script stuff does these days, though it's a bit strange it's using localhost:3000 and working.....since phusion doesn't use that port.  I probably should have just ignored that part of the site since I don't really remember how it works (except that it generated the HTML which we shouldn't ever need to regenerate, hopefully).  @karindalziel might remember more?

Reply:
Cool. I assumed it was probably something not used anymore, but thought I'd ask to understand a little more :)